_includes/profiles.html

```diff
@@ -0,0 +1,53 @@
+{% assign page_category = (include.content | remove: ".html") %}
+<div class="profile_list">
+    {% for category_pair in site.data %}
```

Comment:
Due to the file structure of `_data/...`, traverse through each person's profile was non-standard.